apps/web/tests/digest-playground.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { test, expect } from '@playwright/test';
+
+import { getByTestId, initializeSession } from './utils.ts/browser';
+
+let session;
+
+test.beforeEach(async ({ context }) => {
+  session = await initializeSession(context, { noTemplates: true });
+});
```

Comment:
There is a concept in Playwright called [fixtures](https://playwright.dev/docs/test-fixtures), which we can use to extend the tests with custom functions and refactor this code a little better, but when I wrote this I didn't know about it.